editor/src/templates/editor.tsx

```diff
@@ -706,21 +707,23 @@ export const EditorRoot: React.FunctionComponent<{
 }) => {
   return (
     <AtomsDevtools>
-      <DispatchContext.Provider value={dispatch}>
-        <OriginalMainEditorStateContext.Provider value={mainStore}>
-          <EditorStateContext.Provider value={mainStore}>
-            <DomWalkerMutableStateCtx.Provider value={domWalkerMutableState}>
-              <CanvasStateContext.Provider value={canvasStore}>
-                <LowPriorityStateContext.Provider value={lowPriorityStore}>
-                  <UiJsxCanvasCtxAtom.Provider value={spyCollector}>
-                    <EditorComponent />
-                  </UiJsxCanvasCtxAtom.Provider>
-                </LowPriorityStateContext.Provider>
-              </CanvasStateContext.Provider>
-            </DomWalkerMutableStateCtx.Provider>
-          </EditorStateContext.Provider>
-        </OriginalMainEditorStateContext.Provider>
-      </DispatchContext.Provider>
+      <Provider>
```

Comment:
Maybe we should give the Provider an alias in the import, e.g. `import { Provider as JotaiProvider } from 'jotai'` because we have so many providers here.

Reply:
Great idea, renamed it in https://github.com/concrete-utopia/utopia/pull/4293/commits/faf9be114c8f6d7b7d3dee7f2531a18df4e69977